install_helm.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+USE_SUDO="false"
+HELM_INSTALL_DIR:="./tmp"
```

Comment:
we will need to make sure that the `/tmp` is in the path of the image where the test is running